### PR TITLE
Template Acceptation -> master

### PR DESCRIPTION
**Pull request by YHerm**

### CODE REVIEW

File: src/main/java/frc/Acceptation.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Things To Do When Getting Template
+----------------------------------
+- Power Distribution ID
+- Change field width and length
+- Call swerve update status in robot periodic
```
Comment (hillelv) [inline src/main/java/frc/Acceptation.md:5]: what is swerve? I didn't find it in this branch

**Review by drorta — DISMISSED** [review]

**Review by GilStein1 — DISMISSED** [review]
goofy ahh template

**Review by hillelv — APPROVED** [review]

File: src/main/java/frc/Acceptation.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Things To Do When Getting Template
+----------------------------------
+- Power Distribution ID []
+- Change field width and length []
+- Call swerve update status in robot periodic []
+- Add subsystems to brake manager []
```
Comment (drorta) [inline src/main/java/frc/Acceptation.md:6]: why all the [] in the end?

Comment (YHerm) [reply src/main/java/frc/Acceptation.md:6]: It’s checkboxes

File: src/main/java/frc/Acceptation.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Things To Do When Getting Template
+----------------------------------
+- [ ] Power Distribution ID
+- [ ] Call swerve update status in robot periodic
+- [ ] Add subsystems to brake manager
```
Comment (Sh-Pe) [inline src/main/java/frc/Acceptation.md:5]: I don’t think you need those [ ]

Comment (YHerm) [reply src/main/java/frc/Acceptation.md:5]: u can mark what u did like that

**Pull request merged**